### DISCUSSION TITLE
单个端口能否绑定多个上游服务器组

### DISCUSSION THREAD

**Discussion by EvenLRs** [pymumu/smartdns#1797] [Q&A]
## 背景
如题，为实现各家服务使用各家DNS解析（如阿里的服务使用阿里云DNS解析，腾讯的服务使用DNSPod 公共云DNS解析）本人在SmartDNS上配置了两组Bootstrap DNS。

## 需求
后为实现国内外网站分流，根据互联网上的教程启用了第二DNS，但是在绑定端口的时候，希望能够将两组国内的Bootstrap DNS绑定到同一个端口上。

## 问题
由于未找到相关资料，故来此询问Bind命令是否支持给一个端口同时绑定两个上游服务器组？如可行，能否提供样例命令？如不可行，能否通过输入两条bind命令的形式实现此需求？

**Comment by PikuZheng** [comment]
不能。但是我没理解为啥需要bind两个组，既然这两个组在一个口上同时用，它们的名称为什么不同？
